<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsErrorDlg</class>
 <widget class="QDialog" name="CwmsErrorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fehlerübersicht</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTreeWidget" name="m_pqlvErrors">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Zeit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Schweregrad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modul</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Zeile</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nachricht</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Markierte Fehlermeldung</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="m_pqtText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Datenbankzugriffe über WMS</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Anzahl der Schnittstellenaufrufe:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pqleInterfaceCalls"/>
      </item>
      <item>
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>Anzahl der Nachrichten</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_pqleCompleteMessageCount"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Anzahl der Nachrichten nach Art</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Info</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_pqleInfoMessageCount"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Warning</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="m_pqleWarningMessageCount"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>Error</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_pqleErrorMessageCount"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="textLabel5">
        <property name="text">
         <string>Fatal</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="m_pqleFatalMessageCount"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Einstellungen Sichtbarkeit</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QCheckBox" name="m_pqchbInfo">
        <property name="text">
         <string>Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pqchbWarning">
        <property name="text">
         <string>Warning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pqchbError">
        <property name="text">
         <string>Error</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pqchbFatal">
        <property name="text">
         <string>Fatal</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>348</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_pqpbConfig">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/mainsettings.png</normaloff>:/icons/mainsettings.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbClear">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>320</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="wmsgui.qrc">
         <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="wmsgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>CwmsErrorDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbConfig</sender>
   <signal>clicked()</signal>
   <receiver>CwmsErrorDlg</receiver>
   <slot>ConfigClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqlvErrors</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>CwmsErrorDlg</receiver>
   <slot>ItemSelectedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqchbInfo</sender>
   <signal>clicked()</signal>
   <receiver>CwmsErrorDlg</receiver>
   <slot>SeverityChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>-23</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqchbWarning</sender>
   <signal>clicked()</signal>
   <receiver>CwmsErrorDlg</receiver>
   <slot>SeverityChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>578</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqchbError</sender>
   <signal>clicked()</signal>
   <receiver>CwmsErrorDlg</receiver>
   <slot>SeverityChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqchbFatal</sender>
   <signal>clicked()</signal>
   <receiver>CwmsErrorDlg</receiver>
   <slot>SeverityChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbClear</sender>
   <signal>clicked()</signal>
   <receiver>m_pqlvErrors</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ItemSelectedSlot()</slot>
  <slot>SeverityChangedSlot()</slot>
 </slots>
</ui>
